### ISSUE TITLE
[theme] blockQuote affects only the style of '> ' prefix

### ISSUE THREAD

**Issue by dosmanak** [jaspervdj/patat#111]
Observed behavior: Change of style for blockQuote affects only the first few characters on the line
Expected behavior for theme.blockQuote is to change style of the whole text.

example:
```
theme:
  blockQuote: [italic, vividRed, onDullGreen]
```
![image](https://user-images.githubusercontent.com/6140849/137450400-19d74e92-7304-41fb-8fdd-3fe0dd89c398.png)

```
$ patat --version
0.8.7.0
Using pandoc: 2.11.2
```

**Comment by dosmanak** [comment]
Shall I close the issue?

**Comment by jaspervdj** [comment]
@dosmanak No, your observation is right, I would also like this to change the entire text... I think this should be easier to do now that I did the prettyprinter fix in #128, so I'll have a look.

**Comment by dosmanak** [comment]
:tada:
